### ISSUE TITLE
How about results

### ISSUE THREAD

**Issue by peanut8** [ymfa/seq2seq-summarizer#4]
Good job. How about rouge scores?

**Comment by Colanim** [comment]
I tried to reproduce results reported in the paper [Get to the Point](https://arxiv.org/pdf/1704.04368.pdf), using same parameters.

But I was unsuccessful, results are much lower than expected, the ROUGE scores are half lower than the ones reported in the paper.

Any idea / pretrained model available ?

**Comment by xcfcode** [comment]
@Colanim Have you got good results?

**Comment by astariul** [comment]
Nope, I gave up and tried [this repo,](https://github.com/ChenRocks/fast_abs_rl) which is really easy to reproduce and gave better results anyway.

**Comment by xcfcode** [comment]
@Colanim Thanks a lot!

**Comment by xcfcode** [comment]
@Colanim @ymfa  I have changed something which can help get better results.
@mollynatsu Maybe someone can try to reproduce the results, This is an excellent repo, do not give up.
After which I can get ROUGE-L F1 score about 30

I hope this can help the one who will hold this repo in the future.

- The input of decoder should be (word embedding + t-1 time context vector )


https://github.com/ymfa/seq2seq-summarizer/blob/f6b3de3f179d56537d5badff6d3f950a41eff423/model.py#L127

- The ptr calculation should consider the word embedding

https://github.com/ymfa/seq2seq-summarizer/blob/f6b3de3f179d56537d5badff6d3f950a41eff423/model.py#L171

- Init GRU params

```
    elif isinstance(m, nn.GRU):
        print("Init GRU params...... ")
        for param in m.parameters():
            if len(param.shape) >= 2:
                init.orthogonal_(param.data)
            else:
                init.normal_(param.data)
```

- I use adam with lr equal to 0.001
- Change attention to MLP attention and **create mask**
- norm the loss by traget_len
